tests/cypress.config.ts

```diff
@@ -5,6 +5,7 @@ export default defineConfig({
     failOnStatusCode: false,
     defaultCommandTimeout: 30000,
     videoUploadOnPasses: false,
+    watchForFileChanges: false,
```

Comment:
out of curiosity, is the feature disabled for performance reasons?

Reply:
sorry it's mostly personal preference when executing cypress locally (since intellij auto-saves and executes cypress tests in the middle of changes)

I can remove if it's preferred.

Reply:
all good, that seems actually better this way